tasks/Qc.wdl

```diff
@@ -53,6 +80,7 @@ task CollectAggregationMetrics {
   input {
     File input_bam
     File input_bam_index
+    Boolean is_bam
```

Comment:
NOTE: @tmooney this was a working example of using `ln -s` in a single step. This was tested and of course worked as expected.